modules/cluster-logging-updating-logging-to-5-0.adoc

```diff
@@ -1,41 +1,30 @@
-// Module included in the following assemblies:
-//
-// * logging/cluster-logging-upgrading.adoc
+[id="cluster-logging-updating-logging-to-5-0_{context}"]
+= Updating from cluster logging in {product-title} 4.6 or earlier to OpenShift Logging 5.x
 
-[id="cluster-logging-updating-logging_{context}"]
-= Updating OpenShift Logging
+{product-title} 4.7 made the following name changes:
 
-After updating the {product-title} cluster, you can update from cluster logging 4.6 to Red Hat OpenShift Logging 5.0 by changing the subscription for the OpenShift Elasticsearch Operator and the Cluster Logging Operator.
+* The _cluster logging_ feature became the _Red Hat OpenShift Logging_ 5.x product.
+* The _Cluster Logging_ Operator became the _Red Hat OpenShift Logging_ Operator.
+* The _Elasticsearch_ Operator became _OpenShift Elasticsearch_ Operator.
 
-[NOTE]
-====
-In OpenShift Logging 5.0 and later, the Cluster Logging Operator is called Red Hat OpenShift Logging Operator.
-====
+To upgrade from cluster logging in {product-title} version 4.6 and earlier to OpenShift Logging 5.x, you update the {product-title} cluster to version 4.7 or 4.8. Then, you update the following operators:
 
-When you update:
+* From Elasticsearch Operator 4.x to OpenShift Elasticsearch Operator 5.x
+* From Cluster Logging Operator 4.x to Red Hat OpenShift Logging Operator 5.x
 
-* You must update the OpenShift Elasticsearch Operator before updating the Cluster Logging Operator.
-* You must update both the OpenShift Elasticsearch Operator and the Cluster Logging Operator.
-+
-Kibana is unusable when the OpenShift Elasticsearch Operator has been updated but the Cluster Logging Operator has not been updated.
-+
-If you update the Cluster Logging Operator before the OpenShift Elasticsearch Operator, Kibana does not update and the Kibana custom resource (CR) is not created. To work around this problem, delete the Cluster Logging Operator pod. When the Cluster Logging Operator pod redeploys, the Kibana CR is created.
+You must update the OpenShift Elasticsearch Operator _before_ you update the Red Hat OpenShift Logging Operator. You must also update _both_ Operators to the same version.
```

Comment:
Is this worth an admonition?  (Note, Caution?)

Reply:
Updated. Thanks.